### PR TITLE
DTGB-443: extra articles block

### PR DESCRIPTION
**Pull request by delrueba**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: styleguide/components/31-molecules/teasers/teaser-article/teaser-article.config.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+'use strict';
```
Comment (Jeroen005) [inline styleguide/components/31-molecules/teasers/teaser-article/teaser-article.config.js:1]: Gingen we in stijlgids niet gaan voor Engelstalige voorbeeld content?

**Review by Jeroen005 — APPROVED** [review]

File: styleguide/components/31-molecules/teasers/teaser-article/teaser-article.config.js (hunk 1 above)
Comment (meirege) [reply styleguide/components/31-molecules/teasers/teaser-article/teaser-article.config.js:1]: 👍

File: styleguide/components/71-pages/general/info-page/info-page--variant1.twig [hunk 2]
```diff
@@ -22,37 +22,37 @@
     {% include '@image-gallery--multiple' with {
     "items": [
     {
-    "src": "https://via.placeholder.com/700x432",
+    "src": "https://via.placeholder.com/735x459",
```
Comment (meirege) [inline styleguide/components/71-pages/general/info-page/info-page--variant1.twig:25]: Gebruiken we hier en in de andere pages niet beter echte images? Deze worden namelijk gebruikt ter validatie vanaf volgende week.

**Review by meirege — APPROVED** [review]

File: styleguide/components/31-molecules/teasers/teaser-article/teaser-article.config.js (hunk 1 above)
Comment (delrueba) [reply styleguide/components/31-molecules/teasers/teaser-article/teaser-article.config.js:1]: I'm sorry, didn't quite catch that. Could you use English please?

File: styleguide/components/71-pages/general/info-page/info-page--variant1.twig (hunk 2 above)
Comment (delrueba) [reply styleguide/components/71-pages/general/info-page/info-page--variant1.twig:25]: Updated those in the extra articles block.

**Pull request merged**